content/en_downloads.php

```diff
@@ -48,6 +48,9 @@
       <p>
           Eclipse Kura is available also in a <a href="https://hub.docker.com/r/eclipse/kura/" target="_blank">containerized form</a>.
       </p>
+      <div class="alert alert-warning">
+          Ubi8 based Docker image for Kura 5.2.0 is not available at the moment due a temporary bug with the image buidling process.
```

Comment:
due to a temporary

Reply:
Do we want to put the bug id or the reference to the issue in the ubi8 repo?

Reply:
building process